inference-engine/samples/benchmark_app/CMakeLists.txt

```diff
@@ -5,9 +5,22 @@
 file (GLOB SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 
+find_package(HddlUnite QUIET)
+if(HddlUnite_FOUND)
```

Comment:
My general question relates to exposing this feature to the user. benchmark_app is described in sources, so the user will see code, but there is no information on what is HddlUnite. @ilya-lavrenov